x-pack/legacy/plugins/siem/public/components/ml_popover/popover_description.tsx

```diff
@@ -14,7 +14,7 @@ export const PopoverDescriptionComponent = () => (
   <EuiText size="s">
     <FormattedMessage
       id="xpack.siem.components.mlPopup.anomalyDetectionDescription"
-      defaultMessage="Run any of the Machine Learning jobs below to view anomalous events throughout the SIEM application. We’ve provided a few common detection jobs to get you started. If you wish to add your own custom jobs, simply create and tag them with “SIEM” from the {machineLearning} application for inclusion here."
+      defaultMessage="Run any of the Machine Learning jobs below to prepare for creating signal detection rules that produce signals for detected anomalies, and to view anomalous events throughout the SIEM application. We’ve provided a collection of common detection jobs to get you started. If you wish to add your own custom ML jobs, simply create and tag them with “SIEM” from the {machineLearning} application."
```

Comment:
Thanks! 👍 Also updated to use `group` instead of `tag` as that's the nomenclature within ML.

![image](https://user-images.githubusercontent.com/2946766/78415159-37a4f080-75dd-11ea-8bd5-c9c4688b7747.png)